web-service/helpers.py

```diff
@@ -189,12 +196,8 @@ def GetContentEncoding(content):
             encoding = value[0]
```

Comment:
Sorry Hoa, I didn't see this comment before pulling.

We can revert, but I tested with the known test cases and this so far seems a strict improvement.  Can we do a subsequent patch if its still not perfect?